src/components/Exchange/PositionsList.js

```diff
@@ -84,6 +84,7 @@ export default function PositionsList(props) {
     approvePositionRouter,
```

Comment:
Looks good. This log is a bit hectic but couldnt see any noticeable differences between preview and master? what was changed here

Reply:
I just added trackAction to props thats all, no idea what other changes were made as a result